src/components/TopBar/TopBar.vue

```diff
@@ -4,7 +4,7 @@
 -->
 
 <template>
-	<div class="top-bar" :style="topBarStyle" :data-theme-dark="isInCall">
+	<div class="top-bar" :style="topBarStyle" :data-theme-dark="isInCall ? true : undefined">
```

Comment:
And this is far from `.sync` modifier changes 👀

Reply:
:shrug: